lambda.js

```diff
@@ -523,19 +523,14 @@ stork.status = (event, context, callback) => {
         };
 
         got.get(shaUri, shaConfig)
-          .then(() => {
-            console.log(err);
```

Comment:
I removed the logging of the error here. I believe it's redundant -- it should get ignored if the SHA is just gone. And if the SHA is not gone, it gets thrown by the lambda function when we send it back with `callback(err)` later on.